recipes/shrinkwrap/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -x
+
+cmake \
+  -DBUILD_SHARED_LIBS:BOOL=ON \
+  -DCMAKE_PREFIX_PATH:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+  -DCMAKE_INSTALL_INCLUDEDIR=include \
+  -DCMAKE_INSTALL_LIBDIR=lib \
+  -DCMAKE_BUILD_TYPE=Release \
+  ${SRC_DIR}
+
+make -j${CPU_COUNT}
+make install
```

Comment:
Do we have a `make check` that we can run?

Reply:
@ocefpaf not exactly `make check`, but generated makefile actually provides `make test`. Do you think it's okay to add in this place?

Reply:
Yep. `make test` works :smile:

Reply:
@ocefpaf thanks, added make test and passed.